### ISSUE TITLE
How to add wind vectors over a density heatmap?

### ISSUE THREAD

**Issue by netotz** [plotly/plotly.py#2783]
I have this density heatmap generated using Python Plotly package:

[![enter image description here][1]][1]

Now I need to add wind vectors over the heatmap. I would like to use [quiver plots][2] if possible, but currently I want to know how to add any kind of plot to a mapbox.

I have only found [this example][3] but the source code in the [Plotly Chart Studio][4] doesn't load so I don't know how to do it, also I need it in 2D:

[![enter image description here][5]][5]

I have posted this in [StackOverflow](https://stackoverflow.com/questions/63982915/how-to-add-wind-vectors-over-a-density-heatmap) but I'm afraid of not getting answers 😢 

  [1]: https://i.stack.imgur.com/ZStAC.jpg
  [2]: https://plotly.com/python/quiver-plots/
  [3]: https://blog.plotly.com/post/171585173862/how-to-create-2d-and-3d-interactive-weather-maps
  [4]: https://chart-studio.plotly.com/~empet/14795/wind-speed-forecast/
  [5]: https://i.stack.imgur.com/IkkEo.png

**Comment by RafalDamian** [comment]
Hi, have you figured it out? I have the same problem right now and i cannot find a solution.

**Comment by netotz** [comment]
> 
> 
> Hi, have you figured it out? I have the same problem right now and i cannot find a solution.

no, I had to use a scatter map with arrows to represent the wind

**Comment by aayush-mittal9** [comment]
Can you please share the code where you overlapped the arrows on top of the heatmap? I am facing the same problem btw.

**Comment by netotz** [comment]
@aayush-mittal9 I have it on [this repository](https://github.com/netotz/georreferenciacion) and this is [the code](https://github.com/netotz/georreferenciacion/blob/master/georef/heatmap.py). The comments and docstrings are in Spanish

**Comment by gvwilson** [comment]
Hi - we are tidying up issues and PRs in Plotly's public repositories so that we can focus on things that are most important to our community. For tech support and "how-to" questions, please post in our community forum https://community.plotly.com/. Thank you - @gvwilson
